x/ethbridge/test_common.go

```diff
@@ -0,0 +1,24 @@
+package ethbridge
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/cosmos-sdk/x/bank"
+	"github.com/cosmos/cosmos-sdk/x/supply"
+	"github.com/cosmos/peggy/x/ethbridge/types"
+	oracle "github.com/cosmos/peggy/x/oracle"
+	keeperLib "github.com/cosmos/peggy/x/oracle/keeper"
+)
+
+func CreateTestHandler(t *testing.T, consensusNeeded float64, validatorAmounts []int64) (sdk.Context, oracle.Keeper, bank.Keeper, supply.Keeper, []sdk.ValAddress, sdk.Handler) {
```

Comment:
nit, it'd be better to create a test suite for this: https://godoc.org/github.com/stretchr/testify/suite

Reply:
it'll allow you to also rename this file to `common_test.go`